client/app/scripts/liveblog-edit/components/polls/poll-component-view.tsx

```diff
@@ -0,0 +1,63 @@
+/* eslint camelcase: "off" */
+import React, { useState, useEffect } from 'react';
+import './poll-component.scss';
+import { pollCalculations } from './utils';
+
+export interface IPollBody {
+    active_until: string;
```

Comment:
I suppose the reason for this underscore case it's because it comes from backend?

Reply:
Yes that's the case.